### PR TITLE
addin formik

### PR DESCRIPTION
**Pull request by IbrahimQumseya**

### CODE REVIEW

**Comment by sergiutoma** [comment]
missing-dependencys

**Comment by IbrahimQumseya** [comment]
i think we need just `npm i`

File: src/screens/SignUp.js [hunk 1]
```diff
@@ -11,28 +11,60 @@ import {
...
 function SignUp() {
   const { t } = useTranslation();
   const history = useNavigate();
 
-  const handleSubmit = (e) => {
-    e.preventDefault();
-
-    const data = new FormData(e.currentTarget);
+  const handleSubmit = (values) => {
+    const { email, lastName, firstName, password, passwordConform, username } = values;
```
Comment (sergiutoma) [inline src/screens/SignUp.js:63]: Is it a reason to deconstruct and then rebuild the object(allData)?

**Review by sergiutoma — CHANGES_REQUESTED** [review]
- It should only redirect us to **/login** if the sign-up was successful made.
- The sign-up button should be disabled if terms has not been checked
- The terms button does not open any dialog with the terms to be accepted
- `formik` and `yup` libraries must be installed in the project.(`npm i -S formik yup`)

**Review by sergiutoma — APPROVED** [review]
Looks good! Maybe we can add preview terms in the future. 👍

**Pull request merged**